### PR TITLE
Create login page

### PR DESCRIPTION
**Pull request by tinpham5614**
Related issue: #5 
Related pull request: #6

This is the updated PR following PR #6. Please help to review and feedback, thank you!

### CODE REVIEW

File: app/build.gradle
Comment (Seiyaroo) [inline app/build.gradle:1]: For the compileSDK version are we just maintaining a constant update for that or are we aiming to just stay at base 33? At this stage it probably isn't an issue but we might want to be wary of version upgrades if they potentially break things that are or are not backwards compatible.

**Review by taylorpapke — APPROVED** [review]
Looks great and is easy to read/understand! Everything seems to work fine when I run the code. The login page is visible and I am able to add text into both fields.

**Review by brinkbrink — DISMISSED** [review]
Looks like you've implemented all of the feedback. Thank you, Tin!

**Review by Robel-003 — COMMENTED** [review]
It's working on my end as well. Code is structured and easy to read!

File: app/src/main/res/values/strings.xml
Comment (theGaryLarson) [inline app/src/main/res/values/strings.xml:1]: You've prefixed all your strings with the context (for example, "login"), which is great! It makes it clear where the strings are being used. I recommend continuing this practice with all other strings that you add to your application.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,12 @@
 <resources>
     <string name="app_name">belindas_closet</string>
+    <!--    Login Screen Strings    -->
+    <string name="login_email">Email</string>
+    <string name="login_password">Password</string>
+    <string name="login_title">Resupply</string>
+    <string name="login_create_account">Create a new account</string>
+    <string name="login_button_text">Login</string>
+    <string name="login_logo_description">packaging</string>
```
Comment (theGaryLarson) [inline app/src/main/res/values/strings.xml:9]: I recommend changing this from "packaging" to descriptive text that informs the user that it is a logo.

File: app/src/main/res/values/themes.xml
Comment (theGaryLarson) [inline app/src/main/res/values/themes.xml:1]: Somebody please correct me if this is incorrect but if your app is fully built in Jetpack Compose, you don't need to define a theme in XML; the Kotlin theme (Belindas_closetTheme) is sufficient.

**Review by theGaryLarson — DISMISSED** [review]

File: app/build.gradle [hunk 2]
```diff
@@ -5,12 +5,12 @@ plugins {
 
 android {
     namespace 'com.example.belindas_closet'
```
[the remark this answers is not in the document]
Comment (tinpham5614) [reply app/build.gradle:7]: Thank you for pointing this out. How do we solve this?
I have been reading this [Android Gradle Plugin](https://developer.android.com/build/configure-app-module#set-application-id).

**Comment by tinpham5614** [comment]
Thank you for all your feedback.
I pushed another commit that fixed minor issues as per feedback.
Now, it requires 2 approving reviews.
Please let me know if there is a conflict.
Thank you.

File: app/src/main/res/values/themes.xml
Comment (brinkbrink) [reply app/src/main/res/values/themes.xml:1]: This is definitely something to consider but not necessary for merging.

File: app/build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (brinkbrink) [reply app/build.gradle:7]: Definitely consider this in the future but fine for merging.

**Review by brinkbrink — DISMISSED** [review]
LGTM

**Review by taylorpapke — COMMENTED** [review]
It is saying that there is still a branch conflict that needs to be resolved with the app/build.gradle file. Should we try to troubleshoot this before merging?

**Review by taylorpapke — APPROVED** [review]
Was able to resolve on GH. Resolved to match the code intended from the latest PR. 1 more approval to hopefully get this merged in for good.

**Comment by tinpham5614** [comment]
> Was able to resolve on GH. Resolved to match the code intended from the latest PR. 1 more approval to hopefully get this merged in for good.
Thank you for resolving the problem. I read this [Fix Merge Conflicts](https://www.freecodecamp.org/news/how-to-fix-merge-conflicts-in-git/) but wasn't sure to resolve it.

**Comment by Seiyaroo** [comment]
LGTM

**Review by Seiyaroo — APPROVED** [review]
Putting that final approval through.

**Review by bcko — APPROVED** [review]
Looks good to me. Let's get this pull request merged.

**Pull request merged**